### DISCUSSION TITLE
Main website refinement

### DISCUSSION THREAD

**Discussion by anurag-dhamala** [EddieHubCommunity/support#6894] [Ideas] [answered]
https://www.eddiehub.org/

Hi, I think we can refine EddieHub official website with more modern looks and design. We can add dark mode, add matching color codes and so much more. As someone with experience on web dev, I am willing to contribute. However, I couldn't find the right repo for the website. 

@eddiejaoude 
Do you have any thoughts on this, and can you guide me in the right direction?

**Comment by eddiejaoude** [comment, marked as answer]
Thank you for the suggestion. EddieHub website has been created in a no code tool, so it is not customisable. Have a look at our other pinned repos that are really active and would welcome contributions, for example LinkFree
